CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contributing
+
+Thanks for your interest in making InsightConnect a better security orchestration tool.
+Before you get started, review our [Code of Conduct].
+
+There are multiple ways to help beyond just writing code:
+
+- [Submit bugs and feature requests] with detailed information about your issue or idea.
+- [Help fellow users with open issues] or [help fellow committers test recent pull requests].
+- [Report a security vulnerability in InsightConnect itself or its plugins] to Rapid7.
+- Submit an updated or brand new plugin!  We are always eager for new
+  integrations or features. Don't know where to start? Check out the [developer documentation].
+
+Here's a short list of do's and don'ts to make sure *your* valuable contributions actually make
+it into the master branch.  If you do not care to follow these rules, your contribution
+**will** be closed. Sorry!
+
+## Code Contributions
+
+- **Do** read the [developer documentation]
+- **Do** stick to the [Python PEP8] and [plugin style] guides.
+- **Do** follow the [50/72 rule] for Git commit messages.
+- **Do** license your code as MIT.
+- **Do** create a [topic branch] to work on instead of working directly on `master`. This helps protect the process, ensures users are aware of commits on the branch being considered for merge, allows for a location for more commits to be offered without mingling with other contributor changes, and allows contributors to make progress while a PR is still being reviewed.
```

Comment:
I'm not sure the first part of this statement is necessary. We *should* have merge protection enabled on `master` making it impossible to merge into it without going through a PR process which requires a branch.

Reply:
Done